pkg/app/src/components/OrganisationCard/modules/filtersSection.tsx

```diff
@@ -17,9 +26,20 @@ const useStyles = makeStyles({
 			borderRadius: 8,
 		},
 	},
+	dropdownStyle: {
+		paddingTop: 3,
+		borderRadius: '7%',
+		backgroundColor: PRIMARY_COLOR[3].background,
+		fontSize: 14,
+		'& .MuiMenuItem-root': {
+			paddingBottom: 20,
+			fontSize: 14,
+			minWidth: 200,
+			color: PRIMARY_COLOR[3].text,
```

Comment:
@DarkNebula0 Can you give me some feedback regarding the way I implemented the global theme and whether there is a better way

Reply:
you either 
- use template keys, whenever you have a customisation which is only needed once
- create a custom component with styling in components or
- globally change the styling of a component in the template overrides

usually you would have a global override base style and extend this locally based on your individual case.
hth

Reply:
@developer-sidani when will you fix this?

Reply:
if you mean regarding the font size and border radius we will create fix them in another issue to move on